pkg/minikube/tunnel/route_darwin.go

```diff
@@ -34,6 +36,9 @@ func (router *osRouter) EnsureRouteIsAdded(route *Route) error {
 	if exists {
 		return nil
 	}
+	if err := writeResolverFile(route); err != nil {
+		return fmt.Errorf("could not write /etc/resolver/{cluster_domain} file: %s", err)
```

Comment:
Don't include the path name in the message, as EnsureRouteIsAdded doesn't authoritatively know what it is. My suggestion is: return fmt.Errorf("write resolver file: %v", err)